airflow-core/docs/administration-and-deployment/logging-monitoring/callbacks.rst

```diff
@@ -39,26 +45,34 @@ For example, you may wish to alert when certain tasks have failed, or have the l
 Callback Types
 --------------
 
-There are five types of task events that can trigger a callback:
+There are six types of events that can trigger a callback:
 
 =========================================== ================================================================
 Name                                        Description
 =========================================== ================================================================
-``on_success_callback``                     Invoked when the task :ref:`succeeds <concepts:task-instances>`
-``on_failure_callback``                     Invoked when the task :ref:`fails <concepts:task-instances>`
-``on_retry_callback``                       Invoked when the task is :ref:`up for retry <concepts:task-instances>`
+``on_success_callback``                     Invoked when the DAG or task :ref:`succeeds <concepts:task-instances>`.
+                                            Available at the DAG or task level.
+``on_failure_callback``                     Invoked when the task :ref:`fails <concepts:task-instances>`.
+                                            Available at the DAG or task level.
+``on_retry_callback``                       Invoked when the task is :ref:`up for retry <concepts:task-instances>`.
+                                            Available only at the task level.
 ``on_execute_callback``                     Invoked right before the task begins executing.
+                                            Available only at the task level.
 ``on_skipped_callback``                     Invoked when the task is :ref:`running <concepts:task-instances>` and  AirflowSkipException raised.
                                             Explicitly it is NOT called if a task is not started to be executed because of a preceding branching
                                             decision in the DAG or a trigger rule which causes execution to skip so that the task execution
                                             is never scheduled.
+                                            Available only at the task level.
+``sla_miss_callback``                       Invoked when the DAG misses the defined SLA.
+                                            Available only at the DAG level.
```

Comment:
SLA code has been removed in Airflow 3 - as the hooks are not being called anymore before the new "Alert" feature is established, the parameter is just carried over for compatability.

@ferruzzi probably ou know best, will the callback name be carried over to the new implementation like this or shall we prepare for a new callback - so does it make sense to document the callback state of 2.x here?

Reply:
okay, then the `sla_miss_callback`should be removed. Even if technically in the interface it is not used anymore and is "dead".